requirements/test.txt

```diff
@@ -13,3 +13,5 @@ pre-commit>=1.0
 
 cloudpickle>=1.2
 nltk>=3.3
+requests>=2.24
+aiohttp>=3.6.2
```

Comment:
```suggestion
fsspec[http]
```
wouldn't this work? we can let fsspec handle the requests/aiohttp versions

Reply:
that shall be the same... but lets see

Reply:
seems to be failing the same way...

Reply:
yes, I was not suggesting this to fix anything, but to let fsspec handle the dependencies for http